src/index.js

```diff
@@ -134,7 +134,10 @@ class ReactToPrint extends React.Component {
 
             if (node.sheet) {
               for (let i = 0; i < node.sheet.cssRules.length; i++) {
-                styleCSS += node.sheet.cssRules[i].cssText + "\r\n";
+                //catch 'member not found' error on cssText
+                if (typeof node.sheet.cssRules[i].cssText === "string") {
```

Comment:
Do we need to check that it's a string (can it be anything else?), or just that it exists?

Reply:
Null test didn't work. When it's a string value, it is an acceptable styleCSS. It worked fine with chrome and FF.

Reply:
`if(node.sheet.cssRules[i].cssText)` didn't work?

Out of curiosity, what does IE use instead of this?

Reply:
`if(node.sheet.cssRules[i].cssText)` this catch didn't work. 

In IE11, cssText has an UNKNOWN value when it sees a CSSKeyframesRule with 'transform' property but not a STRING. Chrome and FF has the correct STRING value.

[https://stackoverflow.com/questions/10982739/typeof-returning-unknown-in-ie](https://stackoverflow.com/questions/10982739/typeof-returning-unknown-in-ie )
 _"Basically that’s MS’s answer if you try to test or access something that’s not a true part of their JScript engine."_